.vale/styles/Localization/CommonlyMisusedWords.yml

```diff
@@ -0,0 +1,17 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termswarnings/
+message: 'Only use allow if you are talking about permission. For ability, use enable.'
```

Comment:
I know it's about termswarning, but there is something strange to me.

Either I'm missing something about Vale, either this token list will report everything that contains it.

I'm worried about the fact you will report a warning for something that could be OK in the sentence (something about permission)

Shouldn't you use a `level: suggestion` for this one?

Reply:
Done! Updated as suggested!